src/uct/api/uct.h

```diff
@@ -1205,26 +1198,28 @@ struct uct_ep_params {
     unsigned                            path_index;
 
     /**
-     * Callback is called when the client side endpoint is bound to network
-     * device and provides related info as defined in
-     * @ref uct_cm_ep_resolve_args_t. This field is only valid if
-     * @ref uct_ep_params_t::sockaddr is set.
-     * @note It is never guaranteed that the callaback will be called. If, for
-     * example, the endpoint goes into error state before issuing the connection
-     * request, the callback will not be invoked.
+     * This callback is invoked when the remote server address provided in field
+     * @ref uct_ep_params_t::sockaddr is resolved to the local device to be used
```

Comment:
i think it should be `@ref uct_ep_params_t.sockaddr`

Reply:
why? we use `::` (probably) everywhere. I think doxygen allows both notations.

Reply:
can you pls double-check it?